ile/src/pub_db2.sqlc

```diff
@@ -16,12 +16,12 @@ void check_sql_error(int sqlcode, const char* sqlstate)
 {
   if (sqlcode != 0)
   {
-    DEBUG("SQL Code: %d\n", sqlcode);
-    DEBUG("SQL State: %s\n", sqlstate);
+    DEBUG_WARNING("SQL Code: %d\n", sqlcode);
+    DEBUG_WARNING("SQL State: %s\n", sqlstate);

```

Comment:
If the `sqlcode` is not 0 then this is a problem so it should be an error level message:

```suggestion
    DEBUG_ERROR("SQL Code: %d\n", sqlcode);
    DEBUG_ERROR("SQL State: %s\n", sqlstate);
```